nan.h

```diff
@@ -1393,6 +1393,15 @@ class NanCallback {
     NanDisposePersistent(handle);
   }
 
+  friend bool operator==(NanCallback &cb1, NanCallback &cb2) {
+    if(cb1.IsEmpty() && cb2.IsEmpty()) return 1;
```

Comment:
Also... are you sure these methods are actually called?  It's a member function so the prototype should be:

```c++
bool operator==(const NanCallback& other) const { /* ... */ }
```